tests/integration/test_transposed_metric_log/__init__.py

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env python3
```

Comment:
commented line

Reply:
offcourse :D, I misread it as commented shebang. I noticed that we mostly keep __init__.py file empty, but if its useful we can keep this line.